package.json

```diff
@@ -2,29 +2,24 @@
   "name": "ml-spectra-fitting",
   "version": "4.2.4",
   "description": "Fit spectra using gaussian or lorentzian",
-  "main": "./lib/index.js",
-  "module": "./lib-esm/index.js",
-  "types": "./lib/index.d.ts",
+  "type": "module",
+  "exports": "./lib/index.js",
   "files": [
     "lib",
-    "src",
-    "lib-esm"
+    "src"
   ],
   "scripts": {
-    "build": "npm run tsc && cheminfo-build --entry lib-esm/index.js --root SpectraFitting",
```

Comment:
This is used by the `lactame.yml` workflow and should be kept (adapted with lib/index.js as entry).